src/databrowser.js

```diff
@@ -165,7 +165,7 @@ define([
                 }
                 info = info._parent;
             }
-            vellum.handleDropFinish(target, path);
+            vellum.handleDropFinish(target, path, widget.mug);
```

Comment:
I'm sticking to my line: `widget.mug` should not be passed in here. `vellum.handleDropFinish` uses the mug (i.e., the thing being dropped) to select an operator for the expression editor.

Also, see my comment on core.js.